cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -574,18 +574,22 @@ private void persistAggregation(PostgisAggregator aggregator) throws CygnusPersi
         LOGGER.info("[" + this.getName() + "] Persisting data at NGSIPostgisSink. Schema ("
```

Comment:
Maybe the following to CHANGES_NEXT_RELEASE?

```
[cygnus-ngsi][PostgisSink] Alignement with PostgresqlSink
```

or maybe something a little more detailed about the kind off alignment that has been done in the PR.